CustomCommandExamples/README.md

```diff
@@ -85,6 +87,14 @@ See [Installing DataMiner Integration Studio](https://aka.dataminer.services/Dis
 
 2. The custom command will be executed and the Adaptive Card example should be displayed as the response. <details><summary>`show demo`</summary>![Gif-OutputAnAdaptiveCard](https://user-images.githubusercontent.com/33500507/192817296-cd05a0cb-2267-4639-8d21-32b54bb347cd.gif)</details>
 
+#### Output an Adaptive Card with buttons
```

Comment:
As title casing is used for all the other titles in this file, I'd use it here as well:
```suggestion
#### Output an Adaptive Card with Buttons
```